sql/src/test/java/io/crate/analyze/GroupByAnalyzerTest.java

```diff
@@ -420,6 +442,20 @@ public void testGroupByHavingComplex() throws Exception {
         assertThat(andFunction.arguments().get(1), isFunction("op_not"));
     }
 
+    @Test
+    public void testGroupByHavingComplex2() throws Exception {
```

Comment:
Same thing with these tests here.. `Complex2` has no indication of how this is different from `testGroupByHavingComplex` (which is also a bad test name to begin with).

Keep in mind for test cases: "Could I re-create the test body given the test-case name?" 

If not it is really difficult to reason if a test is still required after a refactoring that would affect this test somehow.